src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -4052,6 +4132,23 @@
     </Field>
     <Field Name="Default" Type="EqualsValueClauseSyntax" Optional="true"/>
   </Node>
+  <Node Name="FunctionPointerParameterSyntax" Base="BaseParameterSyntax">
+    <TypeComment>
+      <summary>Parameter syntax.</summary>
+    </TypeComment>
+    <Kind Name="FunctionPointerParameter"/>
+    <Field Name="AttributeLists" Type="SyntaxList&lt;AttributeListSyntax&gt;" Override="true">
```

Comment:
><Field Name="AttributeLists" Type="SyntaxList&lt;AttributeListSyntax&gt;" Override="true"> [](start = 4, length = 90)

It looks like this field is overridden the same way in both derived nodes. Is there a reason to do the overriding? Would simple inheritance work?

Reply:
Because the generator doesn't generate implementations of the properties in the abstract base class, it leaves them abstract.

---
In reply to: [451217460](https://github.com/dotnet/roslyn/pull/45684#discussion_r451217460) [](ancestors = 451217460)